internal/kibana/policies.go

```diff
@@ -9,30 +9,30 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/pkg/errors"
-
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
 // Policy represents an Agent Policy in Fleet.
 type Policy struct {
-	ID          string `json:"id,omitempty"`
-	Name        string `json:"name"`
-	Description string `json:"description"`
-	Namespace   string `json:"namespace"`
-	Revision    int    `json:"revision,omitempty"`
+	ID                 string   `json:"id,omitempty"`
+	Name               string   `json:"name"`
+	Description        string   `json:"description"`
+	Namespace          string   `json:"namespace"`
+	Revision           int      `json:"revision,omitempty"`
+	MonitoringEnabled  []string `json:"monitoring_enabled"`
+	MonitoringOutputID string   `json:"monitoring_output_id"`
```

Comment:
@marc-gr it looks like this field is not available in old versions of Kibana, it also seems that it is not used here. Was it needed?

Integrations CI is failing for something related to this field https://github.com/elastic/integrations/pull/6673.

Trying to fix it by adding `omitempty` in https://github.com/elastic/elastic-package/pull/1320.

cc @mrodm